src/main/java/it/gov/pagopa/pu/debtpositions/service/DebtPositionTypeOrgServiceImpl.java

```diff
@@ -30,4 +35,13 @@ public IONotificationDTO getIONotificationDetails(Long debtPositionTypeOrgId, Pa
     return null;
   }
 
+  @Transactional
+  @Override
+  public void deleteDebtPositionTypeOrg(Long debtPositionTypeOrgId) {
+    DebtPositionTypeOrg debtPositionTypeOrg = debtPositionTypeOrgRepository.findById(
+      debtPositionTypeOrgId).orElseThrow(
+        () -> new NotFoundException("DebtPositionTypeOrg having id "+debtPositionTypeOrgId+" not found"));
+    debtPositionTypeOrgOperatorsRepository.deleteByDebtPositionTypeOrgId(debtPositionTypeOrgId);
+    debtPositionTypeOrgRepository.delete(debtPositionTypeOrg);
```

Comment:
we should hide the API automatically exposed by SpringDataRest which would delete just the DebtPositionTypeOrg